src/landlab/components/overland_flow/generate_overland_flow_implicit_kinwave.py

```diff
@@ -281,8 +295,11 @@ def runoff_rate(self):
 
     @runoff_rate.setter
     def runoff_rate(self, new_rate):
-        assert new_rate > 0
-        self._runoff_rate = new_rate / 3600000.0  # convert to m/s
+        if isinstance(new_rate, str):
+            self._runoff_rate = self._grid.at_node[new_rate] / 3.6e6
+        else:
+            assert new_rate > 0
```

Comment:
Remove the `assert` statement, it should only be used for debugging. If you want an error to be raised, maybe something like the following,
```python
if new_rate <= 0.0:
    raise ValueError(f"runoff_rate must be positive ({new_rate})")
```